uhal/uhal/include/uhal/ValMem.hpp

```diff
@@ -354,6 +354,12 @@ namespace uhal
       */
       std::size_t size() const;
 
+      /**
+        Return the address of the underlying memory
+        @return the address of the underlying memory
+      */
+      T* data() const;
```

Comment:
Please change the return type to `const T*`  (to match the return type constness of the `std::vector` `data() const` method)